config/vsc_hortense.py

```diff
@@ -226,6 +226,7 @@ def command(self, job):
         {
             'purge_environment': True,
             'resolve_module_conflicts': False,  # avoid loading the module before submitting the job
+            **common_general_config(reframe_prefix)
```

Comment:
`reframe_prefix` is not defined in this config file, as it relies on `$RFM_PREFIX` being set
```suggestion
            **common_general_config()
```